code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -275,38 +275,36 @@ Difficulty: Very Hard
 	max_n_of_items = 200
 	pixel_y = -4
 	use_power = 0
-	var/duplicate = FALSE
 	var/memory_saved = FALSE
 	var/list/stored_items = list()
-	var/list/blacklist = (/obj/item/weapon/spellbook)
+	var/list/blacklist = typecacheof(list(/obj/item/weapon/spellbook))
 
 /obj/machinery/smartfridge/black_box/accept_check(obj/item/O)
-	if(O.type in blacklist)
-		return
-	if(istype(O, /obj/item))
-		return 1
-	return 0
+	if(!istype(O))
```

Comment:
most likely not, but i wasn't gonna bother to dig in smart fridge code to find out.

lest some magic with drag click lets somebody stuff in structures that nobody knew about.